src/main/kotlin/dev/stalla/dom/DomBuilderFactory.kt

```diff
@@ -25,6 +25,7 @@ internal object DomBuilderFactory {
      *
      * See http://bit.ly/owasp-xxe-java-dom (shortened URL on https://cheatsheetseries.owasp.org/)
      */
+    @Suppress("SwallowedException")
```

Comment:
`@Suppress`ed for now. Was thinking that we might want to introduce logging here though in case of the exception, since it has a security impact, and clients should get informed.

Reply:
I don't really think there's a good way to do logging unfortunately, at least not something that can work on Android as well as on JVM. I'd rather suppress and that's it — at any rate, clients aren't really going to act on these things happening at runtime.